tests/integration/index.html

```diff
@@ -18,12 +18,6 @@
       var should = chai.should();
       var assert = chai.assert;
     </script>
-    <script src='deps/pouchdb-1.1.0-postfixed.js'></script>
-    <script src='deps/pouchdb-2.0.0-postfixed.js'></script>
-    <script src='deps/pouchdb-2.2.0-postfixed.js'></script>
-    <script src='deps/pouchdb-3.0.6-postfixed.js'></script>
-    <script src='deps/pouchdb-3.2.0-postfixed.js'></script>
-    <script src='deps/pouchdb-3.6.0-postfixed.js'></script>
     <script src='utils-bundle.js'></script>
```

Comment:
Yup so that's what [these lines](https://github.com/pouchdb/pouchdb/blob/741ac7ba54200548ca54c74aa7c5027786972a6c/tests/integration/browser.migration.js#L44-L57) are doing. The scripts are pulled in asynchronously only when the migration tests are actually run.